client/src/components/FoodSeeker/SearchResults/ResultsFilters/ResultsFilters.js

```diff
@@ -151,10 +152,14 @@ const ResultsFilters = ({
                     onClick={useMyLocationTrigger}
                     disabled={locationPermission === "denied" || !!error}
                     icon="locationSearching"
+					sx={{backgroundColor: palette.common.white}}
```

Comment:
instead of importing palette, you can pass in a function to the `sx` prop and you can grab the theme object

```
sx={(theme) => ({
  backgroundColor: theme.palette.common.white,
})}
```
Also, we clean up all the extra spaces? 🙏

Reply:
Thanks a lot for suggesting to pass in a function to the ```sx``` prop. I corrected the spacing as well. Hope it looks fine now. Thank you.